tsconfig.json

```diff
@@ -9,6 +9,7 @@
 		"declaration": true
 	},
 	"files": [
+    "typings/main.d.ts",
```

Comment:
is this necessary? afaik the core itself doesn't depend on any external typedefinitions...